src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -85,7 +85,7 @@ int MPIDI_OFI_handle_cq_error(int vni, int nic, ssize_t ret);
                               __LINE__,                     \
                               __func__,                       \
                               fi_strerror(-_ret));          \
-        MPIR_ERR_CHKANDJUMP(_retry == 0 && EAGAIN,          \
+        MPIR_ERR_CHKANDJUMP(_retry == 0,                    \
```

Comment:
This is not truly infinite retry any more, I don't think. In the old code, this error condition was never true if the eagain hint was not set. Now it could be true if/when integer math causes `_retry` to hit 0.

Reply:
You mean that `_retry` may wrap around to hit `0`? Indeed, let me rewrite the macro to make it more explicit.